src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 3;
```

Comment:
I think you should start your counter from 0, as you're counting how many times the function has been called. So it wouldn't usually start from 3

Reply:
yep, I agree, the counter should be 0 at the beginning